content/docs/cucumber/api.md

```diff
@@ -848,6 +848,24 @@ The **Command-Line Interface Runner (CLI Runner)** is an executable Java class t
 ```
 java io.cucumber.core.cli.Main
 ```
+Note that you will need to add the `cucumber-core` jar and all of its transitive dependencies to your classpath, in addition to the location of your compiled .class files. ou can find these jars in [Maven Central](https://mvnrepository.com/repos/central). 
```

Comment:
```suggestion
Note that you will need to add the `cucumber-core` jar and all of its transitive dependencies to your classpath, in addition to the location of your compiled .class files. You can find these jars in [Maven Central](https://mvnrepository.com/repos/central). 
```

Reply:
Please fix this typo; you can do so by committing my suggestion (I do not have access to your repo)